components/Footer/src/index.scss

```diff
@@ -0,0 +1,95 @@
+.denhaag-footer {
+  --utrecht-paragraph-color: #fff;
+  --utrecht-paragraph-font-size: 16px;
+  --utrecht-paragraph-line-height: normal;
+  --utrecht-paragraph-margin-block-start: 12px;
+  --utrecht-paragraph-margin-block-end: 12px;
+
+  background-color: #4b4b4b;
+
+  --utrecht-link-social-background-color: transparent;
+  --utrecht-link-social-border-color: transparent;
+  --utrecht-link-social-border-width: 5px;
+  --utrecht-list-social-item-margin-inline-end: 20px;
+  --utrecht-link-social-color: #fff;
+  --utrecht-link-social-hover-color: var(--denhaag-color-ocher-2);
+
+  .utrecht-list-social {
+    align-items: baseline;
+  }
+
+  .denhaag-footer__mega {
+    background-color: var(--denhaag-color-grey-5);
+  }
+
+  .denhaag-list--vertical {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+
+    @media (min-width: 1024px) {
+      grid-template-columns: 1fr 1fr 1fr;
+    }
+
+    @media (min-width: 1280px) {
+      grid-template-columns: unset;
+      grid-auto-flow: column;
+    }
+  }
+
+  .denhaag-link {
+    color: #fff;
+    font-size: 16px;
+    text-decoration-line: none;
+
+    &:hover {
+      text-decoration-line: underline;
+    }
+  }
+
+  .denhaag-footer-legal__container {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: column;
+
+    @media (min-width: 1280px) {
+      flex-direction: row;
+    }
+  }
+
+  .denhaag-list__item:nth-child(1) {
+    padding-inline-start: 0;
+  }
+
+  @media (max-width: 1279px) {
+    .denhaag-list__item {
+      padding-inline-start: 0;
+    }
+  }
+
+  .denhaag-footer__follow-us-list {
+    align-items: baseline;
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+    padding-inline-start: 0;
+
+    @media (min-width: 768px) {
+      justify-content: initial;
+    }
+  }
+
+  .denhaag-footer-group {
```

Comment:
What is a `denhaag-footer-group`? I only see it used once but is it something we will use multiple times? Otherwise we might want to rename it to something like denhaag-footer__contact-container (not the best name, but something like it) :)